devbin/model_extractor/.objs/.gitignore

```diff
@@ -1 +0,0 @@
-model_extractor
```

Comment:
Before removing the `devbin` make system it's probably worth creating the replacement CMake system for the development programs, so that the old system is visible for reference.

Reply:
Yes, I'm currently working on a PR to do that